app/Eloquent/User.php

```diff
@@ -45,4 +46,9 @@ public function profile(): HasOne
     {
         return $this->hasOne(Profile::class);
     }
+
+    public function socialProfiles(): HasMany
```

Comment:
Please remember to add docblock `@property Collection|SocialProfile[] $socialProfiles`.